docs/docs/recipes.md

```diff
@@ -383,6 +383,84 @@ There are so many ways to add styles to your website; Gatsby supports almost eve
 - Use the CSS-in-JS library [Styled Components](/docs/styled-components/)
```

Comment:
We should probably remove the old recipe info, now that the Styling recipes are being added!

Reply:
Sorry I am not sure what should I do here. May you please elaborate?

Reply:
Lines 378-384 can go away, as they're part of the old recipe format.